releasenotes/notes/add-aux-unit-support-curve-analysis-beb4150ef5de283b.yaml

```diff
@@ -0,0 +1,12 @@
+---
+other:
+  - |
+    Tick labels and fit parameters shown in the fit result plot generated by
+    :py:class`qiskit_experiments.curve_analysis.curve_analysis.CurveAnalysis` subclasses
+    are nicely formatted with unit with prefactor scalings.
+    For example, 1e7 might be now displayed as 10 MHz.
+
+    To enable this change, `xval_unit` and `yval_unit` are added to the analysis options of
+    :py:class`qiskit_experiments.curve_analysis.curve_analysis.CurveAnalysis`.
+    These options are likely set by the class itself rather than end-users,
+    and implicitly update the fit plots.
```

Comment:
I agree each experiment module level change is not significant, but base class change should be described in the reno otherwise community people cannot find how to use new functionality in their module (especially we expect people use experiment to develop their library)